### DISCUSSION TITLE
Problema variable color

### DISCUSSION THREAD

**Discussion by JosefinaHevia** [IIC2133-PUC/2024-2#38] [Tarea 0] [answered]
Cuando realizo el make del archivo base main.c sin haberlo modificado (el que entregaron ustedes) me sale:

`clang: warning: -lm: 'linker' input unused [-Wunused-command-line-argument]`

luego si utilizo el primer test easy, en el terminal se escribe:

`Penguin 0 with 6 Puffles:
        Puffle 1735287154 color orange`

Como se puede ver, no se escriben todos lo pinguinos y puffles. A que se puede deber esto?, y como lo corrijo? Quedo atenta a la respuesta.

**Comment by tomastrivino** [comment, marked as answer]
Hola,
Revisa la sugerencia que dejé en #37. Quizás te perjudica el error que hay en el código base respecto a cómo se define el color.
